web/themes/custom/customtheme/templates/block/block--system-menu-block--social-links-footer.html.twig

```diff
@@ -35,7 +35,9 @@
 {% set config = config_pages_field('global_configurations', 'field_social_title', 'default') %}
 {% set heading_id = attributes.id ~ '-menu'|clean_id %}
 <nav role="navigation" aria-labelledby="{{ heading_id }}"{{ attributes|without('role', 'aria-labelledby') }} class="footer-links">
-  <h2{{ title_attributes.addClass('social-title') }}>{{ config[0] }}</h2>
+  {% if config is defined %}
```

Comment:
I'm pretty sure the config is defined in any case

Reply:
> I'm pretty sure the config is defined in any case

<div class="social-title"> does not exist if I deleting global_configurations